gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPluginExtension.groovy

```diff
@@ -65,30 +64,21 @@ public class AndroidCommandPluginExtension {
     // prefer system property over direct setting to enable commandline arguments
     @Memoized
     def getDeviceId() {
-        if (System.properties['deviceId']) {
+        if (System.properties['deviceId'])
```

Comment:
I disagree with removing the curly brackets. It's a risk when refactoring later.